### PR TITLE
Update addDevice welcome page

### PR DESCRIPTION
**Pull request by nmattia**
This updates the "welcome view" of the add device flow to use lit `@click` and the new anchor input component for consistency.

### CODE REVIEW

File: src/frontend/src/flows/login/unknownAnchor.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ const initLinkDevice = (connection: Connection) => {
       "registerUserNumber"
     ) as HTMLInputElement;
 
-    const userNumber = parseUserNumber(userNumberInput.value);
-    await addRemoteDevice(userNumber, connection);
+    const userNumber = parseUserNumber(userNumberInput.value) ?? undefined;
+    await addRemoteDevice(connection, userNumber);
```
Comment (peterpeterparker) [inline src/frontend/src/flows/login/unknownAnchor.ts:149]: note: for these kind of stuffs I find object params for function more reliable `addRemoteDevice({connection, userNumber})` - order of the params becomes irrelevant

File: src/frontend/src/flows/addDevice/welcomeView/index.ts [hunk 2]
```diff
@@ -44,24 +42,15 @@ const pageContent = (userNumber: bigint | null) => html`
  * This shows a prompt to enter the identity anchor to add this new device to.
  */
 export const addRemoteDevice = async (
```
Comment (peterpeterparker) [inline src/frontend/src/flows/addDevice/welcomeView/index.ts:44]: this function doesn't need to be `async` if I am not wrong

**Review by peterpeterparker — APPROVED** [review]
LGTM

File: src/frontend/src/flows/login/unknownAnchor.ts (hunk 1 above)
Comment (nmattia) [reply src/frontend/src/flows/login/unknownAnchor.ts:149]: agreed, but the rest of the codebase doesn't do that and I'd rather stay consistent

**Pull request merged**